starter-code/app.js

```diff
@@ -0,0 +1,67 @@
+Vue.component("bulma-message", {
+  data() {
+    return {
+      showMessage: true
+    }
+  },
+  props: ['header', 'body', 'color'],
+  template: `
+  <article :class="['message', 'is-' + color]">
+    <div class="message-header">
+      <p> {{ header }} </p>
+      <button class="delete" @click="showMessage = !showMessage" aria-label="delete"></button>
+    </div>
+    <div v-show="showMessage" class="message-body">
+      {{ body }}
+    </div>
+  </article>
+  `
+})
+
+Vue.component("bulma-modal", {
+  props: ['head', 'body', 'show'],
+  template: `
+  <div class="modal is-active" v-show="show">
+    <div class="modal-background"></div>
+    <div class="modal-card">
+      <header class="modal-card-head">
+        <p class="modal-card-title"> Country is: {{ head }}</p>
```

Comment:
You probably don't want hardcoded text here, because it makes the component less reusable.